### PR TITLE
Cucumber test for tags

### PR DESCRIPTION
**Pull request by rluck0419**

### CODE REVIEW

File: features/note_tags.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Feature: User tags notes
+
+  Scenario: User creates a note
+    Given I have an existing user account
```
Comment (MattRice12) [inline features/note_tags.feature]: I can show you how to shorten this and reduce repetition in later tests. It looks good otherwise.

**Comment by MattRice12** [comment]
:thumb: 🆙

**Comment by Langster2323** [comment]
👍

File: features/note_tags.feature (hunk 1 above)
Comment (Langster2323) [inline features/note_tags.feature]: ^^Agreed. After observing Matts tests, I can see how this could be reduced. But this is great!

**Pull request merged**